### PR TITLE
Feats : Adds SDIS 07 in partners

### PR DESCRIPTION
**Pull request by fe51**
I would have liked to include the SDIS 07 in our partners as they have supported us since the beginning of the project.

Hence the PR introduces the following modifications : 
- added a third sponsor, SDIS Ardèche
- added its desc, english and french
- updates Atraksis desc accordingly, en and fr

Below the screenshots to see how it renders
- French
image
- English
image

Happy to discuss it !

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -89,7 +89,12 @@ t:
         title: "Atraksis"
         url: "https://atraksis.fr/"
         logo: "https://atraksis.fr/wp-content/themes/atraksis/assets/img/logo-atraksis-footer.svg"
-        content: "Notre partenaire pour les phases de test sur le terrain, et notre intermédiaire avec les pompiers."
+        content: "Acteur de l'innovation pour les secours, c'est notre point de relai avec les pompiers."
+      sponsor3:
+        title: "SDIS Ardeche"
```
Comment (frgfm) [inline _config.yml:94]: Ardèche*

Comment (frgfm) [reply _config.yml:94]: "point de contact"

File: _config.yml [hunk 2]
```diff
@@ -89,7 +89,12 @@ t:
...
+        url: "https://www.sdis07.fr/"
+        logo: "https://www.sdis07.fr/ardeche-secours-incendie/themes/sapeurs-pompiers-ardeche/medias/img/logo-sdis07-pompiers-ardeche.png"
+        content: "Notre partenaire historique, du recueil des besoins aux phases de test sur le terrain."
```
Comment (frgfm) [inline _config.yml:97]: Not sure this is the right way to put it: Atraksis was supporting us way before

Perhaps "Notre partenaire pour le recueil des besoins et l'expérimentation terrain."

File: _config.yml [hunk 3]
```diff
@@ -147,7 +152,12 @@ t:
         title: "Atraksis"
         url: "https://atraksis.fr/"
         logo: "https://atraksis.fr/wp-content/themes/atraksis/assets/img/logo-atraksis-footer.svg"
-        content: "Our partner for field testing phases, and our intermediary with firefighters."
+        content: "An innovation player for rescue services, and our intermediary with firefighters."
```
Comment (frgfm) [inline _config.yml:155]: "A stakeholder for rescue innovation, and our point of contact with firefighters."

File: _config.yml [hunk 4]
```diff
@@ -147,7 +152,12 @@ t:
...
+      sponsor3:
+        title: "SDIS Ardeche"
```
Comment (frgfm) [inline _config.yml:157]: same here

File: _config.yml [hunk 5]
```diff
@@ -147,7 +152,12 @@ t:
...
+        url: "https://www.sdis07.fr/"
+        logo: "https://www.sdis07.fr/ardeche-secours-incendie/themes/sapeurs-pompiers-ardeche/medias/img/logo-sdis07-pompiers-ardeche.png"
+        content: "Our historical partner, from the collection of needs to the field testing phases."
```
Comment (frgfm) [inline _config.yml:160]: same here

**Review by frgfm — COMMENTED** [review]
Thanks! Only a few suggestions, the rest looks good!

File: _config.yml (hunk 2 above)
Comment (fe51) [reply _config.yml:97]: Well, I do not fully agree, we've met Atraksis after and already have engaged discussions with Ardèche. But, I see your point, at we could say both are there from the start so your suggestion suits me :)

**Comment by fe51** [comment]
Suggestions taken into account @frgfm :)

File: _config.yml [hunk 6]
```diff
@@ -147,7 +152,12 @@ t:
         title: "Atraksis"
         url: "https://atraksis.fr/"
         logo: "https://atraksis.fr/wp-content/themes/atraksis/assets/img/logo-atraksis-footer.svg"
-        content: "Our partner for field testing phases, and our intermediary with firefighters."
+        content: "A stakeholder for rescue innovation, and our point of contact with firefighters."
+      sponsor3:
+        title: "SDIS Ardèche"
+        url: "https://www.sdis07.fr/"
+        logo: "https://www.sdis07.fr/ardeche-secours-incendie/themes/sapeurs-pompiers-ardeche/medias/img/logo-sdis07-pompiers-ardeche.png"
+        content: "Our partner for the collection of needs and field experimentation."
```
Comment (frgfm) [inline _config.yml:160]: "collection of needs" unfortunately doesn't translate directly from French :sweat_smile: 

in engineering, usually, it's "requirements elicitation"

**Review by frgfm — COMMENTED** [review]
One slight suggestion for the english translation

File: _config.yml (hunk 6 above)
Comment (fe51) [reply _config.yml:160]: Yes, collection of need is definitely not ok 😅
"Requirements gathering" sounds more common to me ! As you want !

Comment (frgfm) [reply _config.yml:160]: yes, go for it!

**Review by frgfm — APPROVED** [review]

**Review by frgfm — APPROVED** [review]
All good!

**Pull request merged**